base/client.jl

```diff
@@ -71,6 +71,12 @@ function repl_cmd(cmd, out)
         ENV["OLDPWD"] = new_oldpwd
         println(out, pwd())
     else
+        @windows_only begin
+            cmdbuf = IOBuffer()
+            print_cmdshell_escaped(cmdbuf, cmd.exec...)
+            cmd.exec = ["cmd"; "/s /c \"$(bytestring(cmdbuf.data))\""]
```

Comment:
"cmd" should be `joinpath(SHGetKnownFolderPath(FOLDERID_Windows), "cmd.exe")`  (ref #11331), but for now, `joinpath(ENV["WINDIR"], "cmd.exe")` should work.